Connector_USB.pretty/USB_C_Receptacle_Molex_1054500101.kicad_mod

```diff
@@ -0,0 +1,55 @@
+  (fp_line (start -3.4 -0.9) (end 3.4 -0.9) (layer F.CrtYd) (width 0.05))
+  (fp_line (start -3.4 3.9) (end -3.4 -0.9) (layer F.CrtYd) (width 0.05))
+  (fp_line (start -5.2 -4.5) (end -5.2 4.5) (layer F.Fab) (width 0.1))
+  (fp_line (start -5.2 -4.5) (end 5.2 -4.5) (layer F.Fab) (width 0.1))
+  (fp_line (start 5.2 4.5) (end 5.2 -4.5) (layer F.Fab) (width 0.1))
+  (fp_line (start -5.2 4.5) (end 5.2 4.5) (layer F.Fab) (width 0.1))
+  (pad S1 thru_hole oval (at 4.32 -2.67) (size 1.1 2.1) (drill oval 0.6 1.5) (layers *.Cu *.Mask))
+  (pad S1 thru_hole oval (at -4.32 -2.67) (size 1.1 2.1) (drill oval 0.6 1.5) (layers *.Cu *.Mask))
+  (pad S1 thru_hole oval (at -4.32 2.69) (size 1.1 2.6) (drill oval 0.6 2.1) (layers *.Cu *.Mask))
+  (pad S1 thru_hole oval (at 4.32 2.69) (size 1.1 2.6) (drill oval 0.6 2.1) (layers *.Cu *.Mask))
+  (pad B2 smd rect (at 2.25 -1.73) (size 0.3 0.76) (layers F.Cu F.Paste F.Mask))
+  (pad B3 smd rect (at 1.75 -1.73) (size 0.3 0.76) (layers F.Cu F.Paste F.Mask))
+  (pad B4 smd rect (at 1.25 -1.73) (size 0.3 0.76) (layers F.Cu F.Paste F.Mask))
+  (pad B5 smd rect (at 0.75 -1.73) (size 0.3 0.76) (layers F.Cu F.Paste F.Mask))
+  (pad B6 smd rect (at 0.25 -1.73) (size 0.3 0.76) (layers F.Cu F.Paste F.Mask))
+  (pad B7 smd rect (at -0.25 -1.73) (size 0.3 0.76) (layers F.Cu F.Paste F.Mask))
+  (pad B8 smd rect (at -0.75 -1.73) (size 0.3 0.76) (layers F.Cu F.Paste F.Mask))
+  (pad B9 smd rect (at -1.25 -1.73) (size 0.3 0.76) (layers F.Cu F.Paste F.Mask))
+  (pad B10 smd rect (at -1.75 -1.73) (size 0.3 0.76) (layers F.Cu F.Paste F.Mask))
+  (pad B11 smd rect (at -2.25 -1.73) (size 0.3 0.76) (layers F.Cu F.Paste F.Mask))
+  (pad B1 smd rect (at 3.1 -1.73) (size 1 0.76) (layers F.Cu F.Paste F.Mask))
+  (pad B12 smd rect (at -3.1 -1.73) (size 1 0.76) (layers F.Cu F.Paste F.Mask))
+  (pad A12 smd rect (at 3 -3) (size 0.3 0.76) (layers F.Cu F.Paste F.Mask))
+  (pad A11 smd rect (at 2.5 -3) (size 0.3 0.76) (layers F.Cu F.Paste F.Mask))
+  (pad A10 smd rect (at 2 -3) (size 0.3 0.76) (layers F.Cu F.Paste F.Mask))
+  (pad A9 smd rect (at 1.5 -3) (size 0.3 0.76) (layers F.Cu F.Paste F.Mask))
+  (pad A8 smd rect (at 1 -3) (size 0.3 0.76) (layers F.Cu F.Paste F.Mask))
+  (pad A7 smd rect (at 0.5 -3) (size 0.3 0.76) (layers F.Cu F.Paste F.Mask))
+  (pad A6 smd rect (at -0.5 -3) (size 0.3 0.76) (layers F.Cu F.Paste F.Mask))
+  (pad A5 smd rect (at -1 -3) (size 0.3 0.76) (layers F.Cu F.Paste F.Mask))
+  (pad A4 smd rect (at -1.5 -3) (size 0.3 0.76) (layers F.Cu F.Paste F.Mask))
+  (pad A3 smd rect (at -2 -3) (size 0.3 0.76) (layers F.Cu F.Paste F.Mask))
+  (pad A2 smd rect (at -2.5 -3) (size 0.3 0.76) (layers F.Cu F.Paste F.Mask))
+  (pad A1 smd rect (at -3 -3) (size 0.3 0.76) (layers F.Cu F.Paste F.Mask))
+  (model C:/Github/kicad-packages3D/Connector_USB.3dshapes/USB_C_Receptacle_Molex_1054500101.wrl
+    (offset (xyz 0 0 1.5))
+    (scale (xyz 0.4 0.4 0.4))
+    (rotate (xyz 0 0 180))
```

Comment:
That won't work at all.
First of, its a local path that only works on your computer.
Second, the scaling and offset all need to be at default values. See https://kicad-pcb.org/libraries/klc/M2.2/